TESTS/events/timing/main.cpp

```diff
@@ -0,0 +1,122 @@
+#include "mbed_events.h"
```

Comment:
Not a blocker, we should definitely add licenses to our tests (more files are without as I noticed).